server/routes/topic_page.py

```diff
@@ -75,12 +78,20 @@ def topic_page(topic_id=None, place_dcid=None):
     topic_place_config = topic_configs[0]
     topic_place_config.metadata.place_dcid.append(place_dcid)
     # Populuate comparison places (fixed places) for all tiles
+    comparison_places = [place_dcid]
```

Comment:
nit: create a private helper function for getting the topic_place_config for sdg (lines 78-96) because this if case has a lot of specific logic just for sdg

Reply:
Done